### ISSUE TITLE
Web - Bookmark List Page - Missing Heading

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#68]
The text "Bookmarks" should be a heading level one (h1) instead of a heading level four (h4) and the page name/title text should be a heading level two (h2).

Bookmark List - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/bookmarks

**Comment by KyleLivingstonDOL** [comment]
ZTI, can we make this H1 but still have the font size H4? Thanks!
Kyle

**Comment by jfc3-dol** [comment]
By making the item a heading level one (h1), adding a class you can use CSS to make it the current size the heading is now.

**Comment by jfc3-dol** [comment]
There is now a heading at the top of the page but it's a heading level four (h4) and the items below it are heading level three (h3). The page should have on heading level one (h1) which I would expect to be the text "Bookmark" and the headings below (bookmarked items) to be a heading level two (h2).

**Comment by KyleLivingstonDOL** [comment]
ZTI, can you please recode however keep the current font sizes?

**Comment by Abychalil** [comment]
Fixed. Bookmark heading updated to h1 and bookmarked items headings updated to h2. 
![68_Bookmarks page (h1,h2)](https://user-images.githubusercontent.com/49654920/72447078-b3339600-3782-11ea-832e-84a1e49c1a58.PNG)

**Comment by Abychalil** [comment]
Addressed this issue. Please retest and verify whether it meets the required 508 standards.
@jfc3-dol

**Comment by jfc3-dol** [comment]
@aanthonyDOL,
The text "Bookmarks" is now a heading level one (h1), so I'm going to close the issue.

But I'm wondering why only some of the bookmarked items have a heading level two (h2) and others don't is it because only Step items are heading level two (h2) items and the other pages are not? Will check to see if this is listed as an issue already or not and if not create one.
